backend/migrations/0001_initial.py

```diff
@@ -146,6 +146,10 @@ class Migration(migrations.Migration):
                         to=settings.AUTH_USER_MODEL,
                     ),
                 ),
+                (
+                    "main_contact_method",
```

Comment:
Could you please run `py manage.py makemigrations` to create a new migration file rather than editing existing ones. This will break the project for anyone with existing migrations.

Reply:
hey i have been facing this error while trying to run the migration `django.db.utils.OperationalError: no such table: backend_featureflags`

Reply:
Hi @Krishn1412, sorry about the delay I didn't see this comment. Could you update to the latest commit, this should now be fixed. You may need to run `python manage.py migrate` once updated the project. Apologies for the delay